### PR TITLE
ckecli: add etcd local-backup subcommand

### PR DESCRIPTION
**Pull request by ymmt2005**
This command takes backups of CKE-managed etcd.

### CODE REVIEW

**Comment by morimoto-cybozu** [comment]
The added codes and documents look good.
I think that it would be of help to users if the existing documents about the etcd backup files stored in the K8s cluster were updated with a supplementary explanation.
https://github.com/cybozu-go/cke/blob/master/docs/etcd.md#backup
https://github.com/cybozu-go/cke/blob/master/docs/ckecli.md#ckecli-etcd-backup-list
https://github.com/cybozu-go/cke/blob/master/docs/ckecli.md#ckecli-etcd-backup-get-backup_name

**Comment by ymmt2005** [comment]
@morimoto-cybozu 
Addressed in f22297e7fa10b22d04a57dbd20396a7a382cd098 .  PTAL.

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

File: docs/etcd.md [hunk 1]
```diff
@@ -78,86 +78,102 @@ The domain name is `cke-etcd.kube-system.svc.<cluster-domain>`.
 Backup
 ------
 
+There are two ways to take backups of CKE-managed etcd.
+
+1. Use `ckecli etcd local-backup` to take and save a snapshot of etcd locally.
+2. Use `ckecli etcd backup` to configure CronJob to save snapshots in S3-compatible storage periodically.
```
Comment (ymmt2005) [inline docs/etcd.md:84]:
```suggestion
2. Configure CronJob to save snapshots in a filesystem given through a PVC.
```

**Comment by ymmt2005** [comment]
@morimoto-cybozu 
Sorry for bothering you.  I updated the document a bit.
https://github.com/cybozu-go/cke/pull/375/commits/3586adb181fbb2515172d47e764b998a1fa67804

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Pull request merged**